unicodetools/data/ucd/dev/extracted/DerivedLineBreak.txt

```diff
@@ -1430,6 +1430,7 @@ FFED..FFEE    ; AL # So   [2] HALFWIDTH BLACK SQUARE..HALFWIDTH WHITE CIRCLE
 1BC80..1BC88  ; AL # Lo   [9] DUPLOYAN AFFIX HIGH ACUTE..DUPLOYAN AFFIX HIGH VERTICAL
 1BC90..1BC99  ; AL # Lo  [10] DUPLOYAN AFFIX LOW ACUTE..DUPLOYAN AFFIX LOW ARROW
 1BC9C         ; AL # So       DUPLOYAN SIGN O WITH CROSS
+1CEB0..1CEB3  ; AL # So   [4] HORIZONTAL ZIGZAG LINE..BLACK RIGHT TRIANGLE CARET
```

Comment:
I’m guessing AL was chosen based on surrounding blocks but I think this should be ID to match 1F8B2.

Reply:
I forget how I chose it; So is 54% lb=AL and 37% lb=ID without much in the way of obvious patterns.
One pattern I _am_ seeing is that the arrows tend to be lb=**AL**, except for U+1F8B0 and U+1F8B1:
https://util.unicode.org/UnicodeJsps/list-unicodeset.jsp?a=%5B%3Aname%3D%2F%5CbARROW%5Cb%2F%3A%5D&g=lb&i=

 So maybe we should recommend an LB change for those two and make the new one AL?

Looking in the mapping tables at some of the related already-encoded characters that are not otherwise widely used is not that much more enlightening:
- ☞ is ID;
- ⏿ is AL;
- ⦂ is AL.

For their meaning in Smalltalk see the screenshot below from http://www.bitsavers.org/pdf/xerox/parc/techReports/SSL-76-6_Smalltalk-72_Instruction_Manual_Mar76.pdf, p. 49.
![More than you likely wanted to know about Smalltalk-72](https://github.com/unicode-org/unicodetools/assets/2284290/dcbb71e2-3b8b-40c4-a412-345818b75608)

For context, that range contains the following characters:

U+1CEB0 HORIZONTAL ZIGZAG LINE
U+1CEB1 KEYHOLE
U+1CEB2 OLD PERSONAL COMPUTER WITH MONITOR IN PORTRAIT ORIENTATION
U+1CEB3 BLACK RIGHT TRIANGLE CARET

Reply:
OK, considered as operators I will accept AL (or possibly AI).

Reply:
Changed the new arrow to LB=AL; the two ID ones in that block, as well as the block default, will be dealt with separately in #580 if the UTC approves of LB=AL for them.